### PR TITLE
SEAB-6164: resizing and alignment fix for the new landing page

### PR DESCRIPTION
**Pull request by hyunnaye**
**Description**
During the demo, we discovered some tweaks that would improve the landing page. 

Changes made: 

- Changed card text to left align
- The card width was raised to 45% from 40% when aligned into rows to remove the scrollbar on the code block of the Tips and Tricks card
- For window sizes less than `lg` defined [here](https://github.com/angular/flex-layout/blob/master/docs/documentation/Responsive-API.md),, the cards are aligned as columns.
- Removed the horizontal padding in the cards and increased the card content width to 90%

This is the video demo for the resizing

https://github.com/dockstore/dockstore-ui2/assets/61166764/eac1ac29-25a8-445e-8732-f5266adc5f39

**Review Instructions**
Play with the resizing on https://qa.dockstore.org/github-landing-page and verify that things look good.

**Issue**
https://ucsc-cgl.atlassian.net/browse/SEAB-6164

**Security**
If there are any concerns that require extra attention from the security team, highlight them here.

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Check that your code compiles by running `npm run build`
- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.
- [x] If this is the first time you're submitting a PR or even if you just need a refresher, consider reviewing our [style guide](https://github.com/dockstore/dockstore/wiki/Dockstore-Frontend-Opinionated-Style-Guide#pr-checklist)
- [x] Do not bypass Angular sanitization (bypassSecurityTrustHtml, etc.), or justify why you need to do so
- [x] If displaying markdown, use the `markdown-wrapper` component, which does extra sanitization
- [x] Do not use cookies, although this may change in the future
- [x] Run `npm audit` and ensure you are not introducing new vulnerabilities
- [x] Do due diligence on new 3rd party libraries, checking for CVEs
- [x] Don't allow user-uploaded images to be served from the Dockstore domain
- [x] If this PR is for a user-facing feature, create and link a documentation ticket for this feature (usually in the same milestone as the linked issue). Style points if you create a documentation PR directly and link that instead.
- [x] Check whether this PR disables tests. If it legitimately needs to disable a test, create a new ticket to re-enable it in a specific milestone.

### CODE REVIEW

File: src/app/github-landing-page/github-landing-page.component.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h3>Please wait while your GitHub repository installation is being processed. Th
     <hr />
     <h3 class="text-align-center">You can close this tab and return to Dockstore on the original tab.</h3>
 
-    <div class="w-100 cards-container" fxLayout="row" fxLayoutAlign="space-evenly stretch">
```
Comment (kathy-t) [inline src/app/github-landing-page/github-landing-page.component.html:25]: FYI, you could still achieve the resizing by using fxLayout. Something like `fxLayout.lt-sm="column"` which turns the `fxLayout` into a column if the window size is less than "small" (more info here about [sizes](https://github.com/angular/flex-layout/blob/master/docs/documentation/Responsive-API.md))

**Review by kathy-t — COMMENTED** [review]
I feel like the padding inside the cards is quite large, which becomes more obvious when you make the window width really small, shown below. Could we make the text fill the width of the cards more?

image

**Review by kathy-t — APPROVED** [review]

**Review by svonworl — APPROVED** [review]

**Review by denis-yuen — APPROVED** [review]

**Review by coverbeck — COMMENTED** [review]
> The bottom text was not center aligned so that was fixed

I thought you were going to left-align the top text, not center-align the bottom :)

You can probably find links arguing the other way round, but here's [one](https://uxmovement.com/content/why-you-should-never-center-align-paragraph-text/) I found for my position.

If I'm the only who feels this way, I'll concede.

**Comment by hyunnaye** [comment]
> > The bottom text was not center aligned so that was fixed
> 
> I thought you were going to left-align the top text, not center-align the bottom :)
> 
> You can probably find links arguing the other way round, but here's [one](https://uxmovement.com/content/why-you-should-never-center-align-paragraph-text/) I found for my position.
> 
> If I'm the only who feels this way, I'll concede.

Updated the screen recording and alignment. Thanks Charles!

**Review by coverbeck — APPROVED** [review]

**Pull request merged**